### PR TITLE
stored skills in database

### PR DESCRIPTION
**Pull request by chloemiranz**

### CODE REVIEW

File: client/src/app/App.css [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (anthonytate) [inline client/src/app/App.css:1]: Consider leaving out the empty file until code is added and the file is used

File: client/src/app/pages/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, {useState} from 'react';
```
Comment (anthonytate) [inline client/src/app/pages/SignUp.js:1]: Should leave this file out or format it like SignIn.js until the changes are reviewed and submitted on my PR. Currently imports auth-slice which doesn't exist on master.

**Comment by anthonytate** [comment]
It seems like navbar.js and navbar.css shouldn't be included because Navbar.js and Navbar.css are used by App.js and they are pretty much the same files.

File: client/src/app/App.js [hunk 3]
```diff
@@ -1,9 +1,28 @@
...
+import Team from './pages/Team';
+import SignIn from './pages/SignIn';
+import SignUp from './pages/SignUp';
 
 const App = () => (
-  <div className="App">
-    Hello world
-  </div>
+  <Router>
+    <Navbar />
```
Comment (hack-heart) [inline client/src/app/App.js:18]: nit: wrap the JSX inside Router in a <Switch> tag. This allows only the first match to be considered.

File: client/src/app/components/CategoryCard.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
+.category-card {
+    background-color: white;
+    border-radius: 10px;
+    box-shadow: 4px 6px 39px -18px rgba(0,0,0,0.5);
+ }
+  
+ .category-img {
```
Comment (hack-heart) [inline client/src/app/components/CategoryCard.css:7]: alphabetise declarations. See https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules

File: client/src/app/components/CategoryCard.css [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+ .category-title {
+    font-size: 25px;
+    font-weight: 700;
+ }
+  
+ .category-desc {
+    margin-top: 10px;
+    line-height: 24px;
+ }
+
```
Comment (hack-heart) [inline client/src/app/components/CategoryCard.css:31]: nit: no need for the extra lines at the end

File: client/src/app/components/CategoryCard.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import React, { Component, Fragment } from 'react'
```
Comment (hack-heart) [inline client/src/app/components/CategoryCard.js:1]: fragment isn't used. no need to import. Missing semicolon

File: client/src/app/components/CategoryCard.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+ 
+import './CategoryCard.css';
+ 
+class CategoryCard extends Component {
```
Comment (hack-heart) [inline client/src/app/components/CategoryCard.js:5]: component has no state. consider turning into functional component

File: client/src/app/components/CategoryCard.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+   constructor(props) {
+       super(props);
```
Comment (hack-heart) [inline client/src/app/components/CategoryCard.js:7]: spacing here seems more than 2 spaces

File: client/src/app/components/CategoryCard.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+   render() {
+     return (
+       <div className='category-card'>
+         <img className='category-img' src={this.props.img} />
+         <div className='category-text-container'>
+           <div className='category-title'> {this.props.name} </div>
+           <div className='category-desc'> {this.props.desc} </div>
+         </div>
+       </div>
+   )
```
Comment (hack-heart) [inline client/src/app/components/CategoryCard.js:19]: Missing semicolons

File: client/src/app/components/Navbar.js [hunk 10]
```diff
@@ -0,0 +1,21 @@
+import React, { Component, Fragment } from 'react'
```
Comment (hack-heart) [inline client/src/app/components/Navbar.js:1]: No need to import Fragment

File: client/src/app/components/Navbar.js [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+import { NavLink } from 'react-router-dom'
+
+import './Navbar.css'
+
+class Navbar extends Component {
```
Comment (hack-heart) [inline client/src/app/components/Navbar.js:6]: Component is stateless. Consider making it a functional component

File: client/src/app/pages/Explore.js [hunk 12]
```diff
@@ -0,0 +1,101 @@
+import React, { Component, Fragment } from 'react'
+import firebase from '../../firebase';
+
+import './Explore.css';
+
+// Components
```
Comment (hack-heart) [inline client/src/app/pages/Explore.js:6]: nit: no need for comment

File: client/src/app/App.js [hunk 13]
```diff
@@ -1,9 +1,28 @@
...
+    <Route exact path="/" component={Home} />
+    <Route exact path="/explore" component={Explore} />
+    <Route exact path="/team" component={Team} />
+    <Route exact path="/signin" component={SignIn} />
+    <Route exact path="/signup" component={SignUp} />
+    <Route exact path="/subskills" component={SubSkill} />
```
Comment (hack-heart) [inline client/src/app/App.js:24]: nit: what route is /subskills? expect specific skill in url, perhaps 'explore/:skill?' as route?

File: client/src/app/components/navbar.js [hunk 14]
```diff
@@ -0,0 +1,44 @@
...
+                 <NavLink className='nav-link' activeClassName='nav-link-active' exact to='/explore'>Explore</NavLink>
+                 <NavLink className='nav-link' activeClassName='nav-link-active' exact to='/team'>Meet the Team</NavLink>
+                 <NavLink className='nav-link' activeClassName='nav-link-active' exact to='/signin'>Sign In</NavLink>
+             </div>
+         </div>
+       )
+   }
+}
+ 
+export default navbar
```
Comment (hack-heart) [inline client/src/app/components/navbar.js:43]: PLEASE FIX: components should be capitalised

File: client/src/app/pages/Explore.js [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+    this.state = {
+      skillsArr: []
+    }
+  }
+
+  componentDidMount() {
+    const skillsRef = firebase.database().ref('skills');
+    let skillsArr = []
+
+    skillsRef.set({
```
Comment (hack-heart) [inline client/src/app/pages/Explore.js:22]: This seems like a local test or a set up, remove from PR if so

File: client/src/app/pages/Explore.js [hunk 16]
```diff
@@ -0,0 +1,101 @@
...
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      skillsArr: []
+    }
+  }
+
+  componentDidMount() {
+    const skillsRef = firebase.database().ref('skills');
```
Comment (hack-heart) [inline client/src/app/pages/Explore.js:19]: nit: prefer firebase calls in api folder, e.g skills-api.js

File: client/src/app/pages/Explore.js [hunk 17]
```diff
@@ -0,0 +1,101 @@
...
+    
+    skillsRef.once('value').then(snapshot => {
+      snapshot.forEach(keysSnapshot => {
+        var obj = keysSnapshot.val();
+        skillsArr.push(obj);
+      })
+    }).then(() => {
+      console.log(skillsArr);
+      this.setState({
+        skillsArr: skillsArr
```
Comment (hack-heart) [inline client/src/app/pages/Explore.js:73]: nit: consider storing state in redux

File: client/src/app/pages/Explore.js [hunk 18]
```diff
@@ -0,0 +1,101 @@
...
+      }
+    });
+    
+    skillsRef.once('value').then(snapshot => {
+      snapshot.forEach(keysSnapshot => {
+        var obj = keysSnapshot.val();
+        skillsArr.push(obj);
+      })
+    }).then(() => {
+      console.log(skillsArr);
```
Comment (hack-heart) [inline client/src/app/pages/Explore.js:71]: consider removing this

File: client/src/app/pages/Home.js [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+ 
+class Home extends Component {
+   render(){
+       return (
+         <div className='home-container'>
+           <div className='home-text-container'>
+               <div className='home-title'> The first marketplace exclusively for skills</div>
+               <div className='home-subtitle'> Learn, exchange, connect</div>
+               <div className='home-btn-container'>
+                   <a className='home-btn btn-signup' href='./signin' >Sign In</a>
```
Comment (hack-heart) [inline client/src/app/pages/Home.js:13]: Prefer Link tag from react-router-dom to anchor tag. Links do not refresh the page, which is good for performance.

File: client/src/app/pages/SignUp.js [hunk 20]
```diff
@@ -0,0 +1,33 @@
...
+import {connect} from 'react-redux';
```
Comment (hack-heart) [inline client/src/app/pages/SignUp.js:2]: Leave out since part of earlier PR

**Pull request merged**